node/modules/retrieval.go

```diff
@@ -58,3 +70,74 @@ func HandleRetrievalTransports(lc fx.Lifecycle, l *lp2pimpl.TransportsListener)
 		},
 	})
 }
+
+func NewLoadBalancer(cfg *config.Boost) func(h host.Host) (*loadbalancer.LoadBalancer, error) {
+	return func(h host.Host) (*loadbalancer.LoadBalancer, error) {
+		peerConfig := map[peer.ID][]protocol.ID{}
+		if cfg.Dealmaking.BitswapPeerID != "" {
+			bsPeerID, err := peer.Decode(cfg.Dealmaking.BitswapPeerID)
+			if err != nil {
+				return nil, err
+			}
+			peerConfig[bsPeerID] = []protocol.ID{
+				network.ProtocolBitswap,
+				network.ProtocolBitswapNoVers,
+				network.ProtocolBitswapOneOne,
+				network.ProtocolBitswapOneZero,
+			}
```

Comment:
The list of bitswap protocols is repeated in a couple of places, can we move to a constant?